plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/BuildImageParams.java

```diff
@@ -30,16 +32,20 @@
  * @author Alexander Garagatyi
  */
 public class BuildImageParams {
-    // todo add next parameters q, nocache, rm, forcerm
-    private String      repository;
-    private String      tag;
-    private AuthConfigs authConfigs;
-    private Boolean     doForcePull;
-    private Long        memoryLimit;
-    private Long        memorySwapLimit;
-    private List<File>  files;
-    private String      dockerfile;
-    private String      remote;
+    private String             repository;
+    private String             tag;
+    private AuthConfigs        authConfigs;
+    private Boolean            doForcePull;
+    private Long               memoryLimit;
+    private Long               memorySwapLimit;
+    private List<File>         files;
+    private String             dockerfile;
+    private String             remote;
+    private Boolean            q;
```

Comment:
it is not natural to have such a field in Java. Rename it to something meaningful